### DISCUSSION TITLE
Showing Files From Another Users

### DISCUSSION THREAD

**Discussion by akbardwi** [UniSharp/laravel-filemanager#1208] [General]
Hello, I would like to ask about how to view files from other users. This case can only be seen from the administrator and the configuration uses a private folder

**Comment by streamtw** [comment]
There is a document about customizing `private_folder_name` (See [here](https://unisharp.github.io/laravel-filemanager/config#private_folder_name)). But I am sorry to say that this package currently does not provide such detailed permission control you mentioned.
